### ISSUE TITLE
Suggestion to add a basic command to work with Typescript

### ISSUE THREAD

**Issue by paroxyzm** [paralleldrive/riteway#245]
Hello, 
thank you for the lib! Great job!

I'd be interested in trying it out, but I don't know how to make it work with TypeScript.
I'm not so familiar with babel, but I guess it can be done using different script switches in package.json.

It'd be great if you could update the docs, to include just bare-bones info on how to get started with this lib and TS.

Thank you in advance!

**Comment by Creplav** [comment]
Hi @paroxyzm
I had the same issue for a while but I figured it out.
The solution is a little weird but it's just the way that babel works.
Babel needs some extra configuration for the file extensions that cannot be provided in your .babelrc file.

1. Make sure you have `@babel/preset-typescript` installed as a dev dependency.
2. Add it as a preset in your `.babelrc` file.
1. Create a file called `babel-register.js` in your  project.
2. Open the file and write the following code. 
```javascript
const register = require('@babel/register').default;

register({extensions: ['.js', '.ts', '.jsx', '.tsx']});
```
3. In your `package.json` file in the `"test": ""` script require the file you just created:
```json
"test": "nyc riteway -r @babel/polyfill 'src/**/*.test.js' -r './babel-register.js' | tap-nirvana"
```
4. Run `yarn test`

I hope this helps.
This is just one way of doing it. There might be other methods, but this is the first one I came across that worked. 
As an aside, I would not recommend writing your tests in TypeScript. See the article Eric wrote [here](https://medium.com/@_ericelliott/sounds-like-you-want-it-to-do-type-checking-961688f71fb3).
